deploy/helm/kubecf/templates/rbac.yaml

```diff
@@ -0,0 +1,78 @@
+{{- if not .Values.kube.psp.default }}
+---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  annotations:
+    seccomp.security.alpha.kubernetes.io/allowedProfileNames: '*'
+  name: {{ .Release.Name }}-default
+spec:
+  allowPrivilegeEscalation: true
+  allowedCapabilities:
+  - NET_BIND_SERVICE
+  - SYS_ADMIN
+  - SYS_RESOURCE
+  defaultAllowPrivilegeEscalation: true
+  fsGroup:
+    rule: RunAsAny
+  hostPorts:
+  - max: 65535
+    min: 0
+  privileged: true
+  runAsUser:
+    rule: RunAsAny
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  volumes:
+  - configMap
+  - secret
+  - emptyDir
+  - downwardAPI
+  - projected
+  - persistentVolumeClaim
+  - nfs
+  - rbd
+  - cephFS
+  - glusterfs
+  - fc
+  - iscsi
+  - cinder
+  - gcePersistentDisk
+  - awsElasticBlockStore
+  - azureDisk
+  - azureFile
+  - vsphereVolume
+{{- end }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Release.Name }}-default
```

Comment:
I think the Role name (and RoleBinding name below) should be `{{ .Release.Name }}-default-psp` to show that they are about default PSPs. Calling a role just `default` is kind of meaningless.

Reply:
Indeed.

Reply:
Done in https://github.com/SUSE/kubecf/pull/347/commits/7de118a7c5929a7ed23f0e24f79a4b55059a850b.